clients/mobile/app.json

```diff
@@ -12,9 +12,35 @@
       "backgroundColor": "#ffffff"
     },
     "ios": {
-      "supportsTablet": true
+      "supportsTablet": true,
+      "bundleIdentifier": "com.example.mobile",
+      "infoPlist": {
+        "CFBundleURLTypes": [
+          {
+            "CFBundleURLSchemes": [
+              "myapp"
```

Comment:
make the change according to the project name

Reply:
Updated the `CFBundleURLSchemes` to `bloodconnect` as per our project name instead of `myapp`